### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by joyrostie**
Install and examples added

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-80.gearboxjs.preview.cogniteapp.com

File: README.md [hunk 1]
```diff
@@ -7,14 +7,27 @@ Gearbox.js
 [![Build Status](https://travis-ci.org/cognitedata/gearbox.js.svg?branch=master)](https://travis-ci.org/cognitedata/gearbox.js)
 [![codecov](https://codecov.io/gh/cognitedata/gearbox.js/branch/master/graph/badge.svg)](https://codecov.io/gh/cognitedata/gearbox.js)
 
-## Setup
+## Install package
 
-Run `yarn`.
+Run `yarn add @cognite/gearbox` or `npm install @cognite/gearbox --save`.
 
 Install additional dependencies:
```
Comment (f1cognite) [inline README.md]: I would group the install scripts for the dependencies. I don't want to copy&paste 3 times. What about doing `yarn add @cognite/sdk antd styled-components` and the same for npm?

File: README.md [hunk 2]
```diff
@@ -7,14 +7,27 @@ Gearbox.js
...
-2.  Styled-components via `yarn add styled-components` or `npm i styled-components --save`.
+1.  Cognite SDK (JS) `yarn add @cognite/sdk` or `npm install @cognite/sdk --save`
+2.  AntD by running `yarn add antd` or `npm i antd --save`.
+3.  Styled-components via `yarn add styled-components` or `npm i styled-components --save`.
+
+## Setup
+
+Run `yarn`
 
+## Examples
```
Comment (f1cognite) [inline README.md]: Great

File: README.md [hunk 3]
```diff
@@ -7,14 +7,27 @@ Gearbox.js
...
 Install additional dependencies:
-1.  AntD by running `yarn add antd` or `npm i antd --save`.
-2.  Styled-components via `yarn add styled-components` or `npm i styled-components --save`.
+1.  Cognite SDK (JS) `yarn add @cognite/sdk` or `npm install @cognite/sdk --save`
+2.  AntD by running `yarn add antd` or `npm i antd --save`.
+3.  Styled-components via `yarn add styled-components` or `npm i styled-components --save`.
+
+## Setup
+
+Run `yarn`
```
Comment (f1cognite) [inline README.md]: We can remove this part. No-one using the package need to run this (only contributors as ourself) and we know this.

Comment (joyrostie) [reply README.md]: Removed

File: README.md (hunk 1 above)
Comment (joyrostie) [reply README.md]: Please check updated section at the readme branch

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-80.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-80.gearboxjs.preview.cogniteapp.com

**Pull request merged**